test/models/hymod/include/Reservoir_Test.cpp

```diff
@@ -55,47 +62,63 @@ class NonlinearReservoirKernelTest : public ::testing::Test {
     std::vector <std::shared_ptr<Reservoir_Outlet>> ReservoirOutletsVectorMultipleTypes;    
 };
 
-void NonlinearReservoirKernelTest::SetUp() {
+void ReservoirKernelTest::SetUp() {
     
-    setupNoOutletNonlinearReservoir();
+    setupNoOutletReservoir();
+
+    setupNoOutletReservoir2();
 
-    setupOneOutletNonlinearReservoir();
+    setupOneOutletReservoir();
 
-    setupOneOutletHighStorageNonlinearReservoir();
+    setupOneOutletHighStorageReservoir();
 
-    setupMultipleOutletNonlinearReservoir();
+    setupMultipleOutletReservoir();
 
-    setupMultipleOutletOutOfOrderNonlinearReservoir();
+    setupMultipleOutletOutOfOrderReservoir();
 
     setupExponentialOutletReservoir();
 }
 
-void NonlinearReservoirKernelTest::TearDown() {
+void ReservoirKernelTest::TearDown() {
 
 }
 
```

Comment:
Should add some basic test coverage for the add outlet function that was added.  Should be able to add an outlet and assert the number of outlets and the sorted order of activation thresholds

Reply:
@jdmattern-noaa Adding GitHub issue to add more Reservoir unit tests to directly test particular functions in upcoming PRs as discussed with @hellkite500.